modules/blogging/src/Volo.Blogging.Application/Volo/Blogging/Members/MemberAppService.cs

```diff
@@ -0,0 +1,24 @@
+using System;
+using System.Threading.Tasks;
+using Volo.Abp.Domain.Repositories;
+using Volo.Blogging.Posts;
+using Volo.Blogging.Users;
+
+namespace Volo.Blogging.Members;
+
+public class MemberAppService : BloggingAppServiceBase, IMemberAppService
+{
+    private readonly IRepository<BlogUser, Guid> _userRepository;
+
+    public MemberAppService(IRepository<BlogUser, Guid> userRepository)
+    {
+        _userRepository = userRepository;
+    }
+    
+    public async Task<BlogUserDto> GetAsync(string username)
+    {
+        var user = await _userRepository.GetAsync(x => x.UserName == username);
```

Comment:
you must use find... check that  `GetAsync()` returns `null` if no user found in the DB